### ISSUE TITLE
[Feature] Sort lists (Playlist, Search results, etc.) 

### ISSUE THREAD

**Issue by fromSmolsoft** [omega-gg/MotionBox#48]
I hope i haven't overlooked this feature. 
There are multiple list views in the app (search results, playlists, related, etc..) , but lists are "randomly" sorted.  
e.g. lets say there us show where creators releases episodic content but not in playlist (happens on Dailymotion often).  The search-results return mixed order of eps e.g. ep20, ep17, ep01, ep15. 
It would help a lot to just quick sort (no the algorithm :P ) either custom playlist or search results by name or/and upload date.  

Basically change this :
```
MyShow ep20
MyShow ep17
MyShow ep01
MyShow ep15
...
```

to this:
```
MyShow ep01
MyShow ep02
MyShow ep03
MyShow ep04
...
```

or change sort this : 
```
How I found new place      (2022/03/01)
When was it raining inside (2021/01/12)
You are sorting it wrong    (2024/05/06)
...
```
to this:
```
When was it raining inside (2021/01/12)
How I found new place      (2022/03/01)
You are sorting it wrong    (2024/05/06)
...
```

**Comment by 3unjee** [comment]
Greetings @fromSmolsoft, there's no sorting implemented on MotionBox at this moment. The playlist are displayed as is.
